mocks/services/debugger/uploader.go

```diff
@@ -96,3 +96,15 @@ func (mr *MockUploaderIMockRecorder) Start() *gomock.Call {
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Start", reflect.TypeOf((*MockUploaderI)(nil).Start))
 }
+
+// Stop mocks base method.
+func (m *MockUploaderI) Stop() {
+	m.ctrl.T.Helper()
+	m.ctrl.Call(m, "Stop")
+}
+
+// Stop indicates an expected call of Stop.
+func (mr *MockUploaderIMockRecorder) Stop() *gomock.Call {
+	mr.mock.ctrl.T.Helper()
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Stop", reflect.TypeOf((*MockUploaderI)(nil).Stop))
```

Comment:
long line

Reply:
Do we have a convention setup for this? I don't see anything in the guidelines.
Personally, I'm in favour of 125 characters a a limit for more modern display sizes and resolutions.

Reply:
In my editor (GoLand) I think it is set to 120 but yeah I agree that that doesn't seem like a long line.